src/man/include/debug_levels.xml

```diff
@@ -77,6 +77,13 @@
          <emphasis>9</emphasis>,
          <emphasis>0x4000</emphasis>: Extremely low-level tracing information.
     </para>
+    <para>
+         <emphasis>9</emphasis>,
+         <emphasis>0x20000</emphasis>: Performance and statistical data,
```

Comment:
Won't this `level=9` reuse(with different masks) confuse users?

Reply:
Imo, it means that the corresponding messages can be seen either with `debug_level = 9` or if the `0x20000` bit is set, so it is not a reuse but that `debug_level = 9` shows nearly all messages.